package.json

```diff
@@ -27,14 +28,15 @@
     "@bcrs-shared-components/date-picker": "1.2.15",
     "@bcrs-shared-components/document-delivery": "1.2.0",
     "@bcrs-shared-components/effective-date-time": "1.1.15",
-    "@bcrs-shared-components/enums": "1.0.38",
+    "@bcrs-shared-components/enums": "1.0.46",
     "@bcrs-shared-components/genesys-web-message": "1.0.0",
     "@bcrs-shared-components/help-business-number": "1.1.1",
-    "@bcrs-shared-components/interfaces": "1.0.60",
+    "@bcrs-shared-components/interfaces": "1.0.70",
     "@bcrs-shared-components/limited-restoration-panel": "1.0.5",
-    "@bcrs-shared-components/mixins": "1.1.10",
+    "@bcrs-shared-components/mixins": "1.1.20",
     "@bcrs-shared-components/nature-of-business": "1.2.14",
     "@bcrs-shared-components/relationships-panel": "1.0.9",
+    "@bcrs-shared-components/services": "0.0.1",
```

Comment:
Whoops, I should have published this as 1.0.0.

I've just re-published this (as 1.0.3). Please test this as I see an import in fee-services that I don't think will work in a host app (config-helper).
I'll re-publish and let you know. (No point starting off on the wrong foot.)

Reply:
OK, the fee summary seems to work...

![image](https://github.com/bcgov/business-create-ui/assets/10002889/e05a556e-91b2-496a-acec-98d59c398d72)

... but I don't understand how, because it uses "fee-services", which has a "local import" (utils/config-helper), which isn't present in the host app:
https://github.com/bcgov/bcrs-shared-components/blob/ae80cb63b90ecf103496e288ac5a7a960b272855/src/services/fee-services.ts#L3

Not here:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/100c5dd9-3e31-461b-8683-d1f02069cd2b)

Not here, either:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/8093f366-536d-4821-a688-4f8303d8f04c)

Reply:
Sev, isn't the fee summary imported from the sbc-common-components? Shall I change that import to correspond to the one of bcrs-shared-components?

Reply:
Oh, if you're still using the fee summary from the common components then that's how it's working correctly.

Why are you now importing the shared services then?

You _could_ try to use the shared fee summary, but I think it won't work right away, and it's out of scope of this ticket. Up to you.

Reply:
I removed the shared services dependency Sev. Thanks for pointing this out!

Reply:
Resolving.